wordpress-mobile/WordPress-iOS
# Create New WordPress.com site functionality

Removed "Add WordPress.com" TableView Cell and added a new button on the top right of the NavigationBar.

Tests
1. Create WordPress.com site already known exists. Should get "Site already exists" error
2. Create WordPress.com site that doesn't exist. Should create a new site and get sent back to the BlogListViewController
3. Verify "Add Self-hosted Site" works as before

Needs review: @kurzee
